confluent-kafka-mixin/dashboards/confluent-kafka-overview.json

```diff
@@ -298,30 +304,31 @@
         "text": {},
         "textMode": "auto"
       },
-      "pluginVersion": "8.3.3",
+      "pluginVersion": "8.3.5",
       "targets": [
         {
           "datasource": "$datasource",
           "exemplar": true,
-          "expr": "sum(confluent_kafka_server_retained_bytes{scrape_job=~\"$job\",kafka_id=~\"$cluster\",topic=~\"$topic\",partition=~\"$partition\"})/3",
+          "expr": "avg(sum_over_time(confluent_kafka_server_received_records{scrape_job=~\"$job\",kafka_id=~\"$cluster\",topic=~\"$topic\"}[$__rate_interval])/count_over_time(confluent_kafka_server_received_records{scrape_job=~\"$job\",kafka_id=~\"$cluster\",topic=~\"$topic\"}[$__rate_interval])/sum_over_time(confluent_kafka_server_sent_records{scrape_job=~\"$job\",kafka_id=~\"$cluster\",topic=~\"$topic\"}[$__rate_interval])/count_over_time(confluent_kafka_server_sent_records{scrape_job=~\"$job\",kafka_id=~\"$cluster\",topic=~\"$topic\"}[$__rate_interval]))",
+          "hide": false,
           "interval": "",
           "legendFormat": "",
           "refId": "A"
         }
       ],
-      "title": "Retained bytes",
+      "title": "Write/Read Records Percentage",
       "type": "stat"
     },
     {
       "datasource": "$datasource",
-      "description": "The sum of partitions in a cluster. Basic cluster max partitions limit is 2048.",
+      "description": "Basic clusters have a max retained bytes limit of 5 TB prior to replication. ccloud_metrics_retained_bytes is after replication thus needs to be divided by 3.",
```

Comment:
Does confluent cloud allow adjusting the replication factor or is 3 the only allowed value?

Reply:
Good question! I think 3 is the go-to value to give both redundancy and avoid split-brain problems. I don`t think they allow customer to change it for basic instances, but could be possible for dedicated ones, maybe? Will ask Dustin.